runelite-client/src/main/java/net/runelite/client/plugins/customcursor/CustomCursorPlugin.java

```diff
@@ -81,6 +94,15 @@ public void onConfigChanged(ConfigChanged event)
 		}
 	}
 
+	@Subscribe
+	public void onItemContainerChanged(ItemContainerChanged event)
```

Comment:
A player's equipment is not the only item container that exists. (see InventoryID for a list of identified ones) This should check that the changed container was the player's equipment as well.